targets/TARGET_STM/TARGET_STM32H7/TARGET_STM32H747xI/TARGET_PORTENTA_H7/CMakeLists.txt

```diff
@@ -8,7 +8,6 @@ add_library(mbed-portenta-h7-m4 INTERFACE)
 target_sources(mbed-portenta-h7-m4
     INTERFACE
         PeripheralPins.c
-        system_clock_override.c
```

Comment:
Why do you remove this file ?

Reply:
Now the overriding that was done in this file can be done in custom_targets.json, because now the STM32H7 system_clock.c files support a wide range of HSE_VALUEs

Reply:
But you change default configuration for this portenta board ...

Reply:
But, I added new stuff to targets.json that should replicate what this clock override file was doing